lib/src/delegates/asset_picker_builder_delegate.dart

```diff
@@ -1457,14 +1487,20 @@ class DefaultAssetPickerBuilderDelegate
         itemBannedIndicator(context, asset),
       ],
     );
-    return assetGridItemSemanticsBuilder(context, index, asset, content);
+    return assetGridItemSemanticsBuilder(
+      context,
+      index,
+      asset,
+      content,
+      prependItems,
+    );
   }
 
-  int semanticIndex(int index) {
-    if (specialItemPosition != SpecialItemPosition.prepend) {
-      return index + 1;
-    }
-    return index;
+  int semanticIndex(
+    int index,
+    List<SpecialItemResult> prependSpecialItemResults,
```

Comment:
This does not look like a good choice. A public interface should find the balance when providing arguments, here we only provide the prepend items and leave no comments will make users confused.

Reply:
done updating function argument to specialItemModels.